backend/service/aws/aws.go

```diff
@@ -115,7 +115,7 @@ type Client interface {
 	S3StreamingGet(ctx context.Context, region string, bucket string, key string) (io.ReadCloser, error)
 
 	DescribeTable(ctx context.Context, region string, tableName string) (*dynamodbv1.Table, error)
-	UpdateTableCapacity(ctx context.Context, region string, tableName string, targetIndexRcu int64, targetIndexWcu int64) error
+	UpdateTableCapacity(ctx context.Context, region string, tableName string, targetTableRcu int64, targetTableWcu int64) (*dynamodbv1.Status, error)
```

Comment:
i would not return a pointer for status here which is an int

Reply:
Modified to return the status int instead of pointer, and changed `nil, err` returns to `0, err` (Status_UNSPECIFIED) instead